NetKAN/AviationLights.netkan

```diff
@@ -2,25 +2,17 @@
     "spec_version": "v1.4",
     "identifier":   "AviationLights",
     "name":         "Aviation Lights",
-    "author":       [ "MOARdV", "BigNose", "RPGprayer" ],
-    "$kref":        "#/ckan/github/MOARdV/AviationLights",
+    "author":       [ "LisiasT", "MOARdV", "BigNose", "RPGprayer" ],
+    "$kref":        "#/ckan/spacedock/3208",
     "$vref":        "#/ckan/ksp-avc",
+    "x_netkan_force_v": true,
     "x_netkan_epoch": 1,
     "license":      "CC-BY-NC-SA-4.0",
     "resources": {
-        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/173305-*"
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/211781-*"
     },
     "tags": [
         "plugin",
         "parts"
     ],
-    "x_netkan_override": [
-        {
-            "version": "v3.13",
-            "delete": [ "ksp_version" ],
-            "override": {
-                "ksp_version" : "1.2.2"
-            }
-        }
-    ]
```

Comment:
Please leave this override in, it's nice to have for historical purposes

Reply:
fixed on commit https://github.com/KSP-CKAN/NetKAN/pull/9550/commits/838d78dc4fe2afdb6c7fbb8243a8df28cc7ddfb8